public/static/docs/command-reference/import.md

```diff
@@ -201,12 +202,12 @@ $ dvc import git@github.com:iterative/dataset-registry.git \
 ```
 
 `dvc import` provides a better way to incorporate data files tracked in external
-<abbr>DVC repositories</abbr> because it saves the connection between the
-current project and the source project. This means that enough information is
-recorded in an import stage (DVC-file) in order to
-[reproduce](/doc/command-reference/repro) downloading of this same data version
-in the future, where and when needed. This is achieved with the `repo` field,
-for example (matching the import command above):
+Git/DVC repositories because it saves the connection between the current project
```

Comment:
Again, given the context this is specifically speaking about DVC repositories only. Please revert.

Reply:
This has changed. Import now saves the origin `repo` in the .dvc file when importing from a plain old Git repo.

Reply:
Oh right, it talks about data files. Data files should be in DVC. Changing now.